src/main/java/br/com/caelum/carangobom/exception/MarcaNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package br.com.caelum.carangobom.exception;
+
+public class MarcaNotFoundException extends RuntimeException {
+    public MarcaNotFoundException() {
+        super("Marca não encontrado.");
```

Comment:
typozinho maroto hehe
```suggestion
        super("Marca não encontrada.");
```

Reply:
Haha bem observado